linalg/superlu.cpp

```diff
@@ -269,10 +269,10 @@ SuperLUSolver::~SuperLUSolver()
       LUstructFree(LUstruct);
    }
 
-   if ( options->SolveInitialized )
-   {
-      dSolveFinalize(options, SOLVEstruct);
-   }
+   //if ( options->SolveInitialized )
```

Comment:
Will this crash now if this is used before the solve has been initialized?  If we really don't need this check pull out the dead code.

Reply:
It doesn't crash, but I was seeing a memory error through address sanitizer when it checks options->SolveInitialized.